app/views/static_pages/home.html.erb

```diff
@@ -3,6 +3,7 @@
 
   title: @sitehome.headline.content,
   strapline: @sitehome.strapline.content,
+  hero_image: @sitehome.hero_image.asset.thumb.url,
```

Comment:
Could you change it so that the field is called logo instead of hero_image please? It's a bit confusing otherwise, usually a hero image is the large banner image on a landing page, not the logo.

Reply:
Working in another ticket I found I linked what was supposed to be a hero_image as a logo. I will rework again